mopidy_spotify/utils.py

```diff
@@ -33,3 +35,113 @@ def time_logger(name, level=TRACE):
 
 def flatten(list_of_lists):
     return [item for sublist in list_of_lists for item in sublist]
+
+
+class op:
+    def __init__(self, op, tracks, frm, to=0):
+        self.op = op
+        self.tracks = tracks
+        self.frm = frm
+        self.to = to
+
+    def __repr__(self):
+        length = len(self.tracks)
+        first = self.tracks[0].split(":")[-1]
```

Comment:
this is just helpful for debugging; it's not used anywhere (except for printf-debugging). regarding the possibility of an index error, that won't happen, since we never construct op()s with no tracks inside.

if you want, i can remove it completely.

Reply:
If we leave it in, someone will inevitably find a way of calling it without that condition in the future. So we either fix it or remove it. If the debug output is still useful then please fix it.